daemon/cluster/executor/container/container.go

```diff
@@ -374,6 +376,12 @@ func (c *containerConfig) hostConfig() *enginecontainer.HostConfig {
 		}
 	}
 
+	labels := c.task.Networks[0].Network.Spec.Annotations.Labels
+	name := c.task.Networks[0].Network.Spec.Annotations.Name
+
+	if v, ok := labels[naming.PredefinedLabel]; ok && v == "true" && name == "host" {
+		hc.NetworkMode = "host"
```

Comment:
@aboch sure. But if am not wrong, for the default builtin networks, `NetworkMode` are set as `bridge` or `host` or `none` appropriately. Hence I think it is a good idea to set it for all the predefined networks. wdyt ?